hircluster.c

```diff
@@ -859,10 +858,7 @@ dict *parse_cluster_slots(redisClusterContext *cc, redisReply *reply,
                         master = dictGetEntryVal(den);
                         ret = cluster_slot_ref_node(slot, master);
                         if (ret != REDIS_OK) {
-                            __redisClusterSetError(
-                                cc, REDIS_ERR_OOM,
-                                "Slot ref node failed: out of memory.");
-                            goto error;
+                            goto oom;
```

Comment:
cluster_slot_ref_node() is not doing any allocations, and oom is checked on arguments before.
Will change back to `goto error;`